sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb

```diff
@@ -36,5 +45,15 @@ def finish_transaction(transaction, status)
         transaction.finish
       end
     end
+
+    class SentryContextClientMiddleware
+      def call(_worker_class, job, _queue, _redis_pool)
+        return yield unless Sentry.initialized?
+
+        scope = Sentry.get_current_scope
+        job["sentry_user"] = scope.user
```

Comment:
can this check if `scope.user` is presented? adding a `nil` or an empty hash is not necessary.

Reply:
Sure.